server/data/licenceTypes.ts

```diff
@@ -251,8 +251,10 @@ export interface Reporting {
 export interface RiskManagement {
   version?: RiskVersion
   awaitingInformation?: string
+  awaitingOtherInformation?: string
   emsInformation?: string
   emsInformationDetails?: string
+  hasConsideredChecks?: string
```

Comment:
so as a tidy up we could use [discriminated unions](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions) here: 
```
type RiskManagement = RiskManagementV1 | RiskManagementV2

type RiskManagementV1 = {
  version =  1
  ... just v1 fields...
} 

type RiskManagementV2 = {
  version =  2
... just v2 fields
} 
```

as well as providing better docs, then the compiler knows what is going on: 
```
if (riskManagement.version === 1) {
   // can only access the v1 fields 
} else {
  // can only access the v2 fields
}
```

Maybe after the migration?